### PR TITLE
n32: add bevy cheatbook

### PR DESCRIPTION
**Pull request by inodentry**

### CODE REVIEW

File: content/news/032/index.md
Comment (AngelOnFira) [inline content/news/032/index.md:429]: Might be good to add a one-liner of what Bevy Cheatbook is.

```suggestion
The [Bevy Cheatbook] by [Ida Iyes][twitter-iyesgames] is a concise practical reference to the Bevy game engine. In March, many improvements were made.
```

File: content/news/032/index.md
Comment (AngelOnFira) [inline content/news/032/index.md:438]: I feel like the special character for 3 dots might not be great, not sure though

```suggestion
and more...
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
Just a few things :) (feel free to change my wording of the first sentence)

**Comment by inodentry** [comment]
Ah yes ... the classic mistake of "assuming people already know the project and forgetting to introduce it" :grinning: 

OK, please don't merge this right now. I will provide better wording in a few hours when I am at my computer.

File: content/news/032/index.md
Comment (inodentry) [reply content/news/032/index.md:438]: Do you have any reasoning for that?

Usually, I prefer using Unicode punctuation characters rather than "emulating" them with ASCII.

Comment (17cupsofcoffee) [reply content/news/032/index.md:438]: I think it's just for consistency (other people generally don't bother going to the effort of doing the fancy punctuation).

That being said, I just spotted that we can make our static site generator automatically do the ASCII-to-fancy-unicode translation if we want ([`smart_punctuation`](https://www.getzola.org/documentation/getting-started/configuration/) setting), so we might want to look at turning that on at some point - then we'd have consistency no matter how people write their sections.

**Comment by inodentry** [comment]
How about now?

File: content/news/032/index.md
Comment (inodentry) [reply content/news/032/index.md:439]: Ok, I will leave it up to you folks to decide what to do with the punctuation. Feel free to change it if you want to.

Comment (AngelOnFira) [reply content/news/032/index.md:439]: I'm impartial, I don't particularly know if one is better, but I was just thinking along the lines of "ASCII where possible" is likely to not have issues :)

I'm fine to leave this, and talk about it in a separate issue :+1:

**Review by AngelOnFira — APPROVED** [review]

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**
